e2e/testdata/fn-render/invalid-inline-fnconfig/.expected/config.yaml

```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 
 exitCode: 1
-stdErr: 'Error: invalid pipeline: function "gcr.io/kpt-fn/set-label:unstable": functionConfig must be a valid KRM resource with `apiVersion` and `kind` fields'
+stdErr: "functionConfig must be a valid KRM resource with `apiVersion` and `kind` fields"
```

Comment:
Ideally, `kpt` should be able to refer to the `functionConfig` file that has this error, but that will require associating error with the location in `Kptfile` etc and is a bit non-trivial to do. But the previous error contained `function name`, that was helpful in narrow down which `functionConfig` the error is referring to. so may be just prepend `Function image "gcr.io/....":` to the new error message ?

Reply:
The message is in `Reason` of the error output. There is a `Field` value in the output which is a path to the field which has issue. 
```
Field: `pipeline.mutators[1].config`
```
IMO this is more accurate.

Reply:
Oh, my bad, I forgot about the complete CLI out. I thought this is what the user will see. Thanks